src/data/FooterLinks.js

```diff
@@ -0,0 +1,19 @@
+export const footerLinks = {
+    Company: [
+      { label: "Delivery Info", path: "#" },
+      { label: "FAQ", path: "#" },
+      { label: "Support", path: "#" },
+      { label: "Home", path: "#" },
+    ],
+    Template: [
+      { label: "Style Guide", path: "#" },
+      { label: "Changelog", path: "#" },
+      { label: "Licence", path: "#" },
+      { label: "Webflow University", path: "#" },
```

Comment:
Why do you add path to the footerLink structure if you don't use it where you create links?
Could be google.com here and then you will get rid of the condition